src/redis-benchmark.c

```diff
@@ -971,8 +983,13 @@ static void startBenchmarkThreads() {
         pthread_join(config.threads[i]->thread, NULL);
 }
 
-static void benchmark(const char *title, char *cmd, int len) {
+static void benchmark(const char *title, char *cmd, int len, bool readBenchmark) {
```

Comment:
why do we need to state this is a read benchmark? IMO it is more clear if we add a helper method which parse the given command and states if it is read command or not.